pkg/snapshot/snapshottest/snapshot.go

```diff
@@ -5,7 +5,7 @@ you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
 	http://www.apache.org/licenses/LICENSE-2.0
-	
```

Comment:
can we remove the new lines here and all other files.

Reply:
We are maintaining a new line after adding the license URL in all other repos